iceoryx_hoofs/include/iceoryx_hoofs/cxx/expected.hpp

```diff
@@ -234,8 +234,8 @@ class IOX_NO_DISCARD expected<ErrorType> : public FunctionalInterface<expected<E
     template <typename... Targs>
     static expected create_error(Targs&&... args) noexcept;
 
-    /// @brief  returns true if the expected contains an error otherwise false
-    /// @return bool which contains true if the expected contains an error
+    /// @brief  returns true if the expected signals success otherwise false
+    /// @return bool which contains true if the expected signals success
```

Comment:
Could you please change the comment in line 355 as well?

Reply:
Ah, missed that one. Changed now.